app/src/main/java/org/fossasia/phimpme/gallery/adapters/AlbumsAdapter.java

```diff
@@ -75,6 +76,12 @@ public void onBindViewHolder(final AlbumsAdapter.ViewHolder holder, int position
         Album a = SharedMediaActivity.getAlbums().dispAlbums.get(position);
         Media f = a.getCoverAlbum();
 
+        if(a.getPath().contains(Environment.getExternalStorageDirectory().getPath())){
+            holder.storage.setVisibility(View.INVISIBLE);
```

Comment:
Proper formatting should be done for the if statement. Something like the following ,as it is been followed else where as well.
if( ) {
}